deps.edn

```diff
@@ -4,7 +4,7 @@
         io.replikativ/hitchhiker-tree               {:mvn/version "0.2.222"}
         io.replikativ/incognito                     {:mvn/version "0.3.66"}
         io.replikativ/konserve                      {:mvn/version "0.7.275"}
-        persistent-sorted-set/persistent-sorted-set {:mvn/version "0.1.4"}
+        persistent-sorted-set/persistent-sorted-set {:local/root "persistent-sorted-set-0.1.4.jar"} ;; TODO: eventually clojars object
```

Comment:
Good question. I don't know much about how the packaging works. Is there a way to include the jar for now? Otherwise we would need a proper release for our [durability branch of the persistent set](https://github.com/replikativ/persistent-sorted-set/tree/durability-with-pom).

Ideally, having our own fork of the set including the durability layer is only a temporal solution until Nikita has included an implementation in his library. He is currently working it on a [branch of the original persistent set](https://github.com/tonsky/persistent-sorted-set/tree/storage).

Reply:
This is fixed now. As I had mentioned, Nikita already released the persistent-sorted-set a week ago and we can just use his upstream version for now.